rust/kernel/error.rs

```diff
@@ -57,12 +57,28 @@ impl Error {
     pub const EBADF: Self = Error(-(bindings::EBADF as i32));
 
     /// Creates an [`Error`] from a kernel error code.
+    /// when errno given is invalid,
+    /// it will print a warning message and convert it to EINVAL
```

Comment:
Please capitalize the start of a sentence. Also maybe use the passive form. Eg "will be printed" instead of "it will print" and "the error will be converted" instead of "convert it".